nuspec/chocolatey/Wyam.Install.nuspec

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>wyam.install</id>
+    <version>$version$</version>
+    <title>Wyam</title>
+    <authors>Dave Glick</authors>
+    <owners>Dave Glick</owners>
+    <projectUrl>https://wyam.io</projectUrl>
+    <projectSourceUrl>https://github.com/Wyamio/Wyam</projectSourceUrl>
+    <packageSourceUrl>https://github.com/Wyamio/Wyam</packageSourceUrl>
+    <iconUrl>https://wyam.io/Content/images/logo-square-64.png</iconUrl>
+    <licenseUrl>https://github.com/Wyamio/Wyam/blob/master/LICENSE</licenseUrl>
+    <docsUrl>https://wyam.io/docs/</docsUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Wyam is a simple to use, highly modular, and extremely configurable static content generator. This is a tools package that contains the Wyam CLI.</description>
+    <summary>Wyam is a simple to use, highly modular, and extremely configurable static content generator. This is a tools package that contains the Wyam CLI.</summary>
+  <releaseNotes>
+All release notes for Wyam can be found on the GitHub site - https://github.com/Wyamio/Wyam/releases
+  </releaseNotes>
+    <tags>Wyam Static StaticContent StaticSite Blog BlogEngine</tags>
+  </metadata>
+  <files>
+    <file src="..\..\build\**\bin\*" target="tools"/>
+    <file src="..\..\LICENSE" target="LICENSE"/>
+    <file src="VERIFICATION.txt" target="tools"/>
```

Comment:
@phillipsj in the same way that you have moved the LICENSE file to the root of the Chocolatey package, I would also put the VERIFICATION.txt there too.